### PR TITLE
document keeping the registry addr configuration in sync

### PR DESCRIPTION
**Pull request by bparees**
documentation for the configuration issue encountered in https://bugzilla.redhat.com/show_bug.cgi?id=1531500

### CODE REVIEW

**Comment by bparees** [comment]
@mfojtik @dmage ptal

**Comment by dmage** [comment]
It looks like a good recommendation, but I can't confirm that it will fix the problem.

**Comment by bparees** [comment]
> It looks like a good recommendation, but I can't confirm that it will fix the problem.

@mfojtik and I are pretty confident on that front.  Thanks for reviewing it.

@openshift/team-documentation ptal/merge.

File: install_config/master_node_configuration.adoc [hunk 1]
```diff
@@ -344,7 +344,9 @@ permissions.
 |Sets the hostname for the default internal image
 registry. The value must be in `*hostname[:port]*` format.
 For backward compatibility, users can still use `*OPENSHIFT_DEFAULT_REGISTRY*`
-environment variable but this setting overrides the environment variable.
+environment variable but this setting overrides the environment variable.  When
+this is set, the internal registry must its hostname set as well.
```
Comment (bmcelvee) [inline install_config/master_node_configuration.adoc]: s/must its hostname/must set its hostname?

Comment (bparees) [reply install_config/master_node_configuration.adoc]: thanks, "must have its hostname set as well" is what i think intended. fix pushed.

**Comment by bmcelvee** [comment]
Thanks, @bparees! LGTM 🌞

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../install_config/master_node_configuration.adoc#install-config-master-node-configuration[Master and Node Configuration]
|Updated the `InternalRegistryHostname` description in the
xref:../install_config/master_node_configuration.adoc#master-config-image-policy-config[Image Policy Configuration] section.
%
|xref:../install_config/registry/extended_registry_configuration.adoc#install-config-registry-extended-configuration[Extended Registry Configuration]
|Added instructions for keeping the registry addr configuration in sync in the
xref:../install_config/registry/extended_registry_configuration.adoc#setting-the-registry-hostname[Setting the Registry Hostname] and xref:../install_config/registry/extended_registry_configuration.adoc#docker-registry-configuration-reference-openshift[OpenShift] sections.
%

**Pull request merged**